index.html

```diff
@@ -12,31 +12,31 @@
 <body>
     <div class="main">
          <header>
-             <span class="sp1">Ismail</span>
-             <span class="sp2"><img src="./imagees/union.png" alt=""></span>
+             <a href="##"> <span class="sp1">Ismail</span></a>
+             <nav><span class="sp2"><img src="./imagees/Union.png" alt=""></span></nav>
          </header>
          
              <section class="paragraph">
-                <h1 class="text">I'm Ismail  Glad <br> to see you!</h1>
+                <h1 class="text">I'm Ismail  Glad to see you!</h1>
 
                  
                  <p class="text2">
                     I’m a software developer!
-                     I can help you build a product ,
+                     I can help you build a product
                       feature or website Look through
                        some of my work and experience!
                         If you like what you see and 
                         have a project you need coded,
                          don’t hestiate to contact me.
                  </p>
-                 <a href="##">LET’S CONNECT</a>
+                 <h1 class="a">LET’S CONNECT</h1>
```

Comment:
- [ ] To be semantically correct , for best SEO performance & as mentioned by the previous reviewer, you should only have one `<h1>` tag on your page, kindly update `<h1 class="a">LET’S CONNECT</h1>`  to `<h2 class="a">LET’S CONNECT</h2>` 🙏

Reply:
may you please review my changes please

Reply:
please check my changes I have already fixed h1 mistake  in my coddes